modules/indexer/issues/dboptions.go

```diff
@@ -87,8 +85,10 @@ func ToSearchOptions(keyword string, opts *issues_model.IssuesOptions) *SearchOp
 	case "priority", "priorityrepo", "project-column-sorting":
 		// Unsupported sort type for search
 		searchOpt.SortBy = SortByUpdatedDesc
+	case "latest":
+		fallthrough
 	default:
-		searchOpt.SortBy = SortByUpdatedDesc
+		searchOpt.SortBy = SortByCreatedDesc
```

Comment:
Is it alright to change the default block here? My reasoning for changing this was because only invalid sort types would resort in executing the default block (as expected) and users/web handlers should be passing correct sort type anyways. Alternatively I could just add a `case: "" || "latest"` instead of changing this block.

Reply:
I am not sure whether it is right to change the "default" behavior.

I think the old behavior should be kept as-is if it is not clear.

Reply:
Gotcha, I made the necessary change and decided to change the second-to-last case statement to just use a `fallthrough` since it's right above the `default` block and they both do the same thing.